plasmapy/__init__.py

```diff
@@ -18,21 +23,15 @@
     "__citation__",
 ]
 
-# Enforce Python version check during package import.
-# This is the same check as the one at the top of setup.py
 import sys
 
 if sys.version_info < (3, 9):  # coverage: ignore # noqa: UP036
     raise ImportError(
-        f"This version of PlasmaPy does not support Python {sys.version.split()[0]}."
-        "Please upgrade to a newer version."
+        "This version of PlasmaPy does not support Python "
+        f"{sys.version.split()[0]}. Please upgrade to a newer version "
+        "of Python."
     )
 
-# Packages may add whatever they like to this file, but
-# should keep this content at the top.
-# ----------------------------------------------------------------------------
-from importlib.metadata import PackageNotFoundError
```

Comment:
I don't think we need a `PackageNotFoundError` here, since it was not being used in this file.  Running the tests will let us find out!